src/pages/iou/request/step/IOURequestStepConfirmation.tsx

```diff
@@ -381,11 +381,6 @@ function IOURequestStepConfirmation({
                 }
                 continue;
             }
-
-            // Clear category field when the category doesn't exist for selected policy, or it's disabled
-            if (!policyCategories?.[item.category] || !policyCategories[item.category]?.enabled) {
-                setMoneyRequestCategory(item.transactionID, '', policy?.id);
-            }
```

Comment:
Looks like we will be causing https://github.com/Expensify/App/issues/73321 by this change. @cristipaval Can you confirm it is fine to keep the category in the field when that category is deleted and show warning on save.

Reply:
It sounds good to me, though.